src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -26,17 +34,98 @@ private void initMenu() {
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
 
         this.add(this.importMenu);
-        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+        this.add(this.settingsMenu);
+
         // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
+//        this.importMenu.setEnabled(false);
+
+        // Add function to import menu
+        importMenu.addMenuListener(new MenuListener() {
+            @Override
```

Comment:
Maybe we could make a `MenuAdapter` here, so that we don't have to override the methods that we don't need each time.

Reply:
Fix it!